conformance/apis/v1alpha1/profilereport.go

```diff
@@ -23,26 +23,28 @@ type ProfileReport struct {
 	// (e.g. "AdminNetworkPolicy", "BaselineAdminNetworkPolicy")
 	Name string `json:"name"`
 
-	// Core indicates the core support level which includes the set of tests
+	// Standard indicates the standard support level which includes the set of tests
 	// which are the minimum the implementation must pass to be considered at
 	// all conformant.
-	Core Status `json:"core"`
+	Standard Status `json:"standard"`
 
-	// Extended indicates the extended support level which includes additional
+	// Experimental indicates the experimental support level which includes additional
 	// optional features which the implementation may choose to implement
-	// support for, but are not required.
-	Extended *ExtendedStatus `json:"extended,omitempty"`
+	// support for, but are unstable, ungraduated to standard features. However if
+	// implementations are implementing the experimental CR then they are expected to
+	// implement the whole of it.
```

Comment:
Is that true ("expected to implement the whole of it")? That seems bad for implementors; "if you want to try out any of the experimental features, you need to implement all of them before you can release any of them"?

Reply:
I am ok to remove that bit, I add it because I remember you mentioning it -> but on second thoughts maybe I mis-remembered?

Reply:
removed this, I like your suggestion better.

Reply:
We might have talked about that in the context of the "fail closed" PR? And maybe I did say that, but if I did I was wrong. :slightly_smiling_face: